R/Queue.R

```diff
@@ -19,9 +21,9 @@ Queue <- R6::R6Class( # nolint
     #' @description
     #' Adds element(s) to `Queue`.
     #'
-    #' @param new_elements vector of elements to add
+    #' @param new_elements vector of elements to add.
     #'
-    #' @return self invisibly
+    #' @return Self, invisibly.
```

Comment:
```suggestion
    #' @return `self`, invisibly.
```
Can you check this other places as well.

Reply:
I'm on the fence on this one as it's not a data type.

I won't push it though, I'll make the change

Reply:
there was discussion on this https://github.com/insightsengineering/teal.slice/pull/506#discussion_r1464750879. in r6 documentation as well its used with backticks

Reply:
Done in f20642a3